src/utils_latency.c

```diff
@@ -126,8 +126,8 @@ latency_counter_t *latency_counter_create (void) /* {{{ */
     return (NULL);
   memset (lc, 0, sizeof (*lc));
 
-  latency_counter_reset (lc);
   lc->bin_width = HISTOGRAM_DEFAULT_BIN_WIDTH;
+  latency_counter_reset (lc);
```

Comment:
latency_counter_reset() has 'cdtime_t max_bin = (lc->max - 1) / lc->bin_width;' so it would be division by zero without this change.